app/src/main/java/com/rkm/tasky/util/validator/implementation/NamePatternValidator.kt

```diff
@@ -4,8 +4,8 @@ import com.rkm.tasky.util.validator.abstraction.Validator
 import javax.inject.Inject
 
 class NamePatternValidator @Inject constructor(): Validator {
+    private val namePattern = "^[\\p{L}\\s-]{4,50}$".toRegex()
     override fun validate(input: String): Boolean {
-        val namePattern = "^[A-Za-z\\s-]{4,50}$".toRegex()
         return input.isNotEmpty() && namePattern.matches(input)
     }
```

Comment:
Does this account for special characters common in other languages like ê, ç and á?
As long as the name meets the length requirement and does not include any digits, it should be valid, no?

Reply:
Yes it does